### PR TITLE
SONAR-7626 Deprecate issues report (html/console) feature

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: sonar-scanner-engine/src/main/java/org/sonar/batch/scan/report/ConsoleReport.java [hunk 1]
```diff
@@ -98,6 +98,7 @@ public void setNoFile(boolean value) {
   @Override
   public void execute() {
     if (settings.getBoolean(CONSOLE_REPORT_ENABLED_KEY)) {
+      LOG.info("Console report is deprecated. Use SonarLint CLI to have local reports of issues");
```
Comment (henryju) [inline sonar-scanner-engine/src/main/java/org/sonar/batch/scan/report/ConsoleReport.java]: I would log with WARN level

File: sonar-scanner-engine/src/main/java/org/sonar/batch/scan/report/HtmlReport.java [hunk 2]
```diff
@@ -77,6 +77,7 @@ public HtmlReport(Settings settings, FileSystem fs, IssuesReportBuilder builder,
   @Override
   public void execute() {
     if (settings.getBoolean(HTML_REPORT_ENABLED_KEY)) {
+      LOG.info("HTML report is deprecated. Use SonarLint CLI to have local reports of issues");
```
Comment (henryju) [inline sonar-scanner-engine/src/main/java/org/sonar/batch/scan/report/HtmlReport.java]: Same comment

**Comment by henryju** [comment]
LGTM but I would log with WARN level

**Comment by dbmeneses** [comment]
alright, changing to warn

**Pull request merged**